src/forms/select-styles.js

```diff
@@ -138,6 +138,12 @@ const customStyles = {
   }),
   multiValue: (provided, { isDisabled }) => ({
     ...provided,
+    // Long multiple values truncating ~18chars
```

Comment:
No styling changes should be necessary. No matter how it bad the component looks, it was created to match pf3 design and we should not change them.

Reply:
So I just should remove the styling?

Reply:
yup. Selec is styled as it should be. If you have some styling issues in the tagging component, you should create some local override. This should not be applied to the default styles.